raiden-network/light-client
# Feature/dapp add reset connection button

**Thank you for submitting this pull request :)**

Fixes #2690

**Short description**
Builds on top of #2691 I'll rebase on it as soon as this PR got merged.
This is most interesting for the WalletConnect provider

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Run the dApp and connect via WalletConnect (needs according configuration file for RPC endpoint and no MetaMask)
2. Stop the dApp
3. Close the session in the wallet app
4. Return to the dApp and try to connect again
5. Verify that it does not show the QR code but tries to get the subkey signature by the wallet
6. Wait until the connection pending dialog pops up
7. Click the button to reset the connection
8. After the page got reloaded try to connect again and make sure the QR code shows again
